examples/navier-stokes/advection.h

```diff
@@ -200,19 +200,19 @@ static int Advection(void *ctx, CeedInt Q,
     // -- Total Energy
     // ---- Version 1: dv E u
     if (1) {
-      dv[i+(4+5*0)*Q]  = E*(u[0]*wBJ[0] + u[1]*wBJ[1] + u[2]*wBJ[2]);
-      dv[i+(4+5*1)*Q]  = E*(u[0]*wBJ[3] + u[1]*wBJ[4] + u[2]*wBJ[5]);
-      dv[i+(4+5*2)*Q]  = E*(u[0]*wBJ[6] + u[1]*wBJ[7] + u[2]*wBJ[8]);
+      dv[i+(4+5*0)*Q] = E*(u[0]*wBJ[0] + u[1]*wBJ[1] + u[2]*wBJ[2]);
+      dv[i+(4+5*1)*Q] = E*(u[0]*wBJ[3] + u[1]*wBJ[4] + u[2]*wBJ[5]);
+      dv[i+(4+5*2)*Q] = E*(u[0]*wBJ[6] + u[1]*wBJ[7] + u[2]*wBJ[8]);
       v[i+4*Q] = 0;
     }
     // ---- Version 2: v E du
     if (0) {
       dv[i+(4+0*5)*Q] = 0;
       dv[i+(4+1*5)*Q] = 0;
       dv[i+(4+2*5)*Q] = 0;
-      v[i+4*Q]   = E*(du[0]*wBJ[0] + du[3]*wBJ[1] + du[6]*wBJ[2]);
-      v[i+4*Q]  -= E*(du[1]*wBJ[3] + du[4]*wBJ[4] + du[7]*wBJ[5]);
-      v[i+4*Q]  -= E*(du[2]*wBJ[6] + du[5]*wBJ[7] + du[8]*wBJ[8]);
+      v[i+4*Q] = - (E*(du[0]*wBJ[0] + du[3]*wBJ[1] + du[6]*wBJ[2]) +
+                    E*(du[1]*wBJ[3] + du[4]*wBJ[4] + du[7]*wBJ[5]) +
+                    E*(du[2]*wBJ[6] + du[5]*wBJ[7] + du[8]*wBJ[8]);
```

Comment:
Looks like `E` may as well come outside.

Reply:
Right. Looking at the latest version, I think we very well might have done a copy-paste when we first did version 2 as a debugging check, left it pretty similar to version 1, and then never cleaned it up after adding the `if(0)`.